crates/bevy_app/src/app_builder.rs

```diff
@@ -259,11 +363,47 @@ impl AppBuilder {
         self
     }
 
+    /// Sets the main runner loop function for bevy application
+    ///
+    /// Usually the main loop is handled by bevy integrated plugins (`winit`), but
+    /// in some cases one wants to implement an own main loop.
+    ///
+    /// This method sets the main loop function. Overwrites previous runner.
+    ///
+    /// You should call `app.update()` in the runner to trigger the bevy ecs system.
+    ///
+    /// ## Example
+    /// ```
+    /// use bevy_app::prelude::*;
+    ///
+    /// fn my_runner(mut app: App) {
+    ///     loop {
+    ///         println!("In main loop");
+    ///         app.update();
+    ///     }
+    /// }
+    ///
+    /// App::build()
+    ///     .set_runner(my_runner);
+    /// ```
     pub fn set_runner(&mut self, run_fn: impl Fn(App) + 'static) -> &mut Self {
         self.app.runner = Box::new(run_fn);
         self
     }
 
+    /// Adds a single plugin
+    ///
+    /// One of Bevy's core principles is modularity. All bevy engines are implemented
```

Comment:
I don't think engines is the right word.

Reply:
Oops, missed the "features" part on "bevy engine features"